### PR TITLE
buyer locality apartment model dao service added

### PR DESCRIPTION
**Pull request by SainathB**
buyer locality apartment model dao service added

### CODE REVIEW

File: src/main/java/com/foodybuddy/dao/impl/ApartmentDAOImpl.java [hunk 1]
```diff
@@ -8,38 +8,94 @@
...
+     * @return apartment 
+     */
+	public Apartment getById(Integer id) throws NullPointerException {
+	    if(id == null) {
+            throw new NullPointerException("null id is passed to Apartment getById");
+        }
 		String query = "FROM Apartment WHERE id = " + id;
 		Apartment apartment = (Apartment)session.createQuery(query).uniqueResult();
 		return apartment;
 	}
```
Comment (junzy) [inline src/main/java/com/foodybuddy/dao/impl/ApartmentDAOImpl.java:100]: What is the reason for removing delete function? for instance How will cleanup be done during testing

File: src/main/java/com/foodybuddy/dao/impl/BuyerDAOImpl.java [hunk 2]
```diff
@@ -8,37 +8,89 @@
...
+        }
 		session.update(buyer);
+		return buyer;
 	}
+	
+	/***
+	 * Buyer delete
+	 * @param buyer
+	 * @return buyer
+	 * @throws NullPointerException
```
Comment (junzy) [inline src/main/java/com/foodybuddy/dao/impl/BuyerDAOImpl.java:62]: remove commented out code

File: src/main/java/com/foodybuddy/dao/impl/BuyerDAOImpl.java [hunk 3]
```diff
@@ -8,37 +8,89 @@
...
-	public Buyer getById(int id) {
+	
+	/***
+	 * Buyer getById
+	 * @param id
+	 * @return buyer
+	 * @throws NullPointerException
+	 */
+	public Buyer getById(Integer id) throws NullPointerException {
 		String query = "From Buyer WHERE id = " + id;
```
Comment (junzy) [inline src/main/java/com/foodybuddy/dao/impl/BuyerDAOImpl.java:91]: throws null pointer ex, but doesnt check for it?

File: src/main/java/com/foodybuddy/dao/impl/ApartmentDAOImpl.java [hunk 4]
```diff
@@ -8,38 +8,94 @@
...
+	public Apartment delete(Apartment apartment) throws NullPointerException {
+	    if(apartment == null) {
+            throw new NullPointerException("null apartment is passed to Apartment delete");
+        }
+        session.delete(apartment);
+        return apartment;
+	}
+	*/
+	
+	/***
```
Comment (sairamch04) [inline src/main/java/com/foodybuddy/dao/impl/ApartmentDAOImpl.java:76]: Proper formatting is missing for comments 
/**\* doesn't indent with next line

File: src/main/java/com/foodybuddy/service/BuyerService.java [hunk 5]
```diff
@@ -0,0 +1,18 @@
+package com.foodybuddy.service;
+
```
Comment (sairamch04) [inline src/main/java/com/foodybuddy/service/BuyerService.java:2]: JavaAuto doc is missings.

**Pull request closed without merging**